patches/minecraft/net/minecraft/client/Minecraft.java.patch

```diff
@@ -352,20 +380,21 @@
 -            }
 -
 -         }
++         if (!net.minecraftforge.client.ForgeHooksClient.onClickInput(2, this.field_71474_y.field_74322_I, null).isCanceled())
```

Comment:
You're better off just putting MAIN_HAND instead of null, otherwise you should put `@Nullable`  on the getHand method. The problem is that though is every modder will have to create null check when it could simply be avoid completely by putting MAIN_HAND there instead.